src/backend/src/lib.rs

```diff
@@ -165,6 +165,14 @@ fn post_upgrade(arg: Option<Arg>) {
     }
 }
 
+/// Show the canister configuration.
+#[query(guard = "caller_is_allowed")]
+#[allow(clippy::needless_pass_by_value)]
+#[must_use]
```

Comment:
Other API endpoints do not specify `must_use` except `http_request`. Should we extend all methods with it or should it be removed here?

Reply:
Thanks for the exaplanation. Ok then, let's ensure that when we add new endpoints we also use `#[must_use]` and someday let's add it to existing functions.